x-pack/plugin/eql/qa/common/src/main/resources/additional_test_queries.toml

```diff
@@ -268,10 +268,10 @@ sequence by unique_pid
   [any where true]
   [any where serial_event_id < 72]
 '''
-expected_event_ids  = [54, 55, 59,
+expected_event_ids  = [16, 60, 66,
+                       54, 55, 59,
                        55, 59, 61,
                        59, 61, 65,
-                       16, 60, 66,
+                       60, 66, 71,
                        61, 65, 67,
-                       65, 67, 70,
-                       60, 66, 71]
+                       65, 67, 70]
```

Comment:
Previously sequences were added based on their insertion order which is essentially the order of the last events in a sequence.
This can create some unexpected results as you can see on the left: [16,60,66] occurs after [55,59,61] and [59,61,65]. That is [16,60] occur after [59,61] because it's the last event that counts.
I've changed this to be the order of the first event (not last one) in the sequence since this is what is seen first.